CMakeLists.txt

```diff
@@ -696,6 +696,12 @@ else()
     set(LD_SECURITY_FLAGS "${LD_SECURITY_FLAGS} -Wl,-z,noexecheap")
   endif()
 
+  if(BACKCOMPAT)
+      add_definitions(-DFDELT_TYPE=long\ int)
+      add_linker_flag_if_supported(-Wl,--wrap=__divmoddi4 LD_SECURITY_FLAGS)
+      add_linker_flag_if_supported(-Wl,--wrap=log2f LD_SECURITY_FLAGS)
```

Comment:
Is this really about security ?

Reply:
I wanted to discuss this a bit. I added them to the security flags, because it's just another linker command. I'm fine with calling them LD_COMPAT_FLAGS instead.
I don't really want to use them in the first place, but there is a choice. Either we use the linker flags, like for divmoddi4 here , or we don't and remove the symbol entirely, like I did with glob in https://github.com/monero-project/unbound/pull/12 .

Reply:
Renamed.